contracts/ExternalBribe.sol

```diff
@@ -305,25 +359,45 @@ contract ExternalBribe is IBribe {
         emit NotifyReward(msg.sender, token, adjustedTstamp, amount);
     }
 
-    function swapOutRewardToken(uint i, address oldToken, address newToken) external {
-        require(msg.sender == IVotingEscrow(_ve).team(), 'only team');
+    function swapOutRewardToken(
+        uint256 i,
+        address oldToken,
+        address newToken
+    ) external {
+        require(msg.sender == IVotingEscrow(_ve).team(), "only team");
         require(rewards[i] == oldToken);
         isReward[oldToken] = false;
         isReward[newToken] = true;
         rewards[i] = newToken;
     }
 
-    function _safeTransfer(address token, address to, uint256 value) internal {
+    function _safeTransfer(
+        address token,
+        address to,
+        uint256 value
+    ) internal {
         require(token.code.length > 0);
-        (bool success, bytes memory data) =
-        token.call(abi.encodeWithSelector(IERC20.transfer.selector, to, value));
+        (bool success, bytes memory data) = token.call(
+            abi.encodeWithSelector(IERC20.transfer.selector, to, value)
+        );
         require(success && (data.length == 0 || abi.decode(data, (bool))));
     }
 
-    function _safeTransferFrom(address token, address from, address to, uint256 value) internal {
+    function _safeTransferFrom(
+        address token,
+        address from,
+        address to,
+        uint256 value
+    ) internal {
         require(token.code.length > 0);
-        (bool success, bytes memory data) =
-        token.call(abi.encodeWithSelector(IERC20.transferFrom.selector, from, to, value));
+        (bool success, bytes memory data) = token.call(
```

Comment:
@duncan4123 all the formatting and uint -> uint256 should not have been in this PR or at least in a separate PR, it is very difficult to review with the amount of changes that don't actually matter